### DISCUSSION TITLE
Import path aliases not working using turbo repo and nextjs

### DISCUSSION THREAD

**Discussion by fernandortec** [vercel/turborepo#7824] [Help]
### Summary

Whenever I try to import a file from inside a package using relative paths ("@/component/file"), I get this error:
![image](https://github.com/vercel/turbo/assets/62437672/9d9f1c1a-4408-4bae-86a7-bdc1e4b68bea)

I'm only able to import the module if I use absolute paths ("../../component/file"), my vscode doesn't throw any errors, and I can't seem to find a solution for this. I'll paste all my config below

--------------------

Path => packages => server => src => actions => login.ts

![image](https://github.com/vercel/turbo/assets/62437672/6be9db3d-ccfb-44ac-9876-9128e220bd0b)

my tsconfig.json

```
{
	"extends": "@repo/typescript-config/base.json",
	"compilerOptions": {
		"paths": {
			"@/*": ["./src/*"],
		},
		"baseUrl": ".",
		
		"declaration": false,
		"declarationMap": false
	},
	
	"include": ["src"],
	"exclude": ["node_modules", "dist"]
}

```
my base config.json

```
{
	"$schema": "https://json.schemastore.org/tsconfig",
	"display": "Default",
	"compilerOptions": {
		"lib": ["ESNext"],
		"target": "ESNext",
		"module": "ESNext",
		"declaration": true,
		"declarationMap": true,
		"esModuleInterop": true,
		"incremental": false,
		"isolatedModules": true,
		"moduleDetection": "force",
		"moduleResolution": "Bundler",
		"noUncheckedIndexedAccess": true,
		"resolveJsonModule": true,
		"skipLibCheck": true,
		"strict": true,
		"allowImportingTsExtensions": true,
		"verbatimModuleSyntax": true,
		"noEmit": true
	}
}

```

and the package.json of this package

```
{
	"name": "@repo/server",
	"version": "0.0.1",
	"dependencies": {
		"@repo/schemas": "workspace:*",
		"@neondatabase/serverless": "^0.9.0",
		"@paralleldrive/cuid2": "^2.2.2",
		"bcrypt-ts": "^5.0.2",
		"drizzle-orm": "^0.30.4",
		"postgres": "^3.4.4"
	},
	"exports": {
		".": "./src/index.ts",
		"./use-cases": "./src/use-cases.ts",
		"./db": "./src/db.ts",
		"./actions": "./src/actions.ts"
	},
	"devDependencies": {
		"drizzle-kit": "^0.20.14"
	}
}

```

### Example

https://github.com/fernandortec/next-auth-v5

**Comment by anthonyshew** [comment]
Duplicate of https://github.com/vercel/turbo/discussions/620

**Reply by fernandortec** [reply]
It's not a duplicate, in #620, they're trying to import a package into the apps folder, and I'm trying to import files inside my packages, it's all happening inside the packages, and the solution there, unfortunately, doesn't apply for me
